toucan_connectors/redshift/redshift_connector.py

```diff
@@ -0,0 +1,97 @@
+from contextlib import suppress
+from typing import Dict
+
+import pandas as pd
+import redshift_connector
```

Comment:
Cet import ne va pas marcher, car ce script s'appelle aussi `redshift_connector`  (du coup on ce message dans l'IDE: `Import resolves to its containing file`  
Une solution pourrait être de renommer le connecteur en autre chose, par exemple, `redshift_database_connector.py`